diff --git a/src/tensor_conversion.cpp b/src/tensor_conversion.cpp
index 9aae67caf..20f82162e 100644
--- a/src/tensor_conversion.cpp
+++ b/src/tensor_conversion.cpp
@@ -51,8 +51,10 @@ static int getMatTypeFromTensorPrecision(ovms::Precision tensorPrecision) {
         return CV_32F;
     case ovms::Precision::FP64:
         return CV_64F;
+//  TODO @atobisze - deosn't work on MP external/ovms/src/tensor_conversion.cpp:55:16: error: 'CV_16F' was not declared in this scope; did you mean 'CV_16S'?
     case ovms::Precision::FP16:
-        return CV_16F;
+        //  return CV_16F;
+        return -1;
     case ovms::Precision::I16:
         return CV_16S;
     case ovms::Precision::U8:
